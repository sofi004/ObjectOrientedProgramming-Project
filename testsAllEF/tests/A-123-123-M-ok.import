linhas=41
colunas=32
1;1|'123
1;2|'456
1;3|'abc
1;4|'def
2;1|25
2;2|43
2;3|=1;1
1;5|=2;3
3;4|'123
3;5|=3;4
6;2|'123
4;1|=1;5
4;3|'123
4;2|'123
10;10|'123
8;2|'123
