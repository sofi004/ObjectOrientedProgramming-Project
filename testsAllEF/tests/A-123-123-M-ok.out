Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: Valor a procurar: 1;1|'123
1;5|'123=2;3
2;3|'123=1;1
3;4|'123
3;5|'123=3;4
4;1|'123=1;5
4;2|'123
4;3|'123
6;2|'123
8;2|'123
10;10|'123
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 